src/firefox/remote.js

```diff
@@ -174,8 +174,8 @@ export class RemoteFirefox {
     const addon = await this.getInstalledAddon(addonId);
     await this.checkForAddonReloading(addon);
     await this.addonRequest(addon, 'reload');
-    log.info(
-      `${(new Date()).toTimeString()}: Reloaded extension: ${addon.id}`);
+    process.stdout.write(`${'\r' +
+          'Last extension reload: '}${ (new Date()).toTimeString()}`);
```

Comment:
Please remove this patch since it has its own pull request. Try to always create a separate branch (based on master) for each change. Here's the kind of workflow I recommend: https://yangsu.github.io/pull-request-tutorial/